pkg/api/v1/types.go

```diff
@@ -1720,6 +1720,9 @@ type OpenstackNodeSpec struct {
 	// Max time to wait for the instance to be ready, i.e. 10s/1m
 	// required: false
 	InstanceReadyCheckTimeout string `json:"instanceReadyCheckTimeout"`
+	// UUID of the server group, used to configure affinity or anti-affinity of the VM instances relative to hypervisor
+	// required: false
+	ServerGroup string `json:"serverGroup"`
```

Comment:
Should this (and the struct field later on) be tagged `omitempty` maybe, so it's not included in JSON if it's not set?

Reply:
It wouldn't really matter since in both cases the field is initialized as empty in the MachineDeployment.